example/src/screens/Examples/AwareScrollViewStickyFooter/StickyFooter.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { memo, useMemo, type ReactNode } from 'react';
+import { ViewProps, type ViewStyle } from 'react-native';
+import { useReanimatedKeyboardAnimation } from 'react-native-keyboard-controller';
+import Animated, {
+  interpolate,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+export interface StickyFooterProps extends Pick<ViewProps, 'onLayout'> {
+  children: ReactNode;
+  /**
+   * The offset to add when the keyboard is closed. Useful to position the
+   * sticky bar above things such as a tab bar.
+   */
+  bottomOffsetWhenKeyboardIsClosed?: number;
+}
+
+export const StickyFooter = memo(
+  ({
+    children,
+    bottomOffsetWhenKeyboardIsClosed,
+    ...props
+  }: StickyFooterProps) => {
+    const { height, progress } = useReanimatedKeyboardAnimation();
+    const { bottom } = useSafeAreaInsets();
+
+    const actionBarAnimatedStyles = useAnimatedStyle(() => {
+      const translateY = interpolate(progress.value, [0, 1], [0, height.value]);
+
+      /**
+       * When the keyboard opens, we don't want additional padding since things
+       * like tab bar and safe area are not visible.
+       */
+      const paddingBottom = interpolate(
+        progress.value,
+        [0, 1],
+        [bottomOffsetWhenKeyboardIsClosed ?? 0, 0]
+      );
+
+      return {
+        paddingBottom,
+        transform: [{ translateY }],
+      };
+    }, [bottom, height, progress]);
+
+    const actionBarStyles = useMemo<ViewStyle[]>(
+      () => [
+        {
+          position: 'absolute',
```

Comment:
Do I correctly understand, that `paddingBottom` is needed because we specify `absolute` position and it has an overlap with system UI?

If we don't use `absolute` positioning (as offered below) - will we need to have this `paddingBottom`.

Anyway, I'm not excluding, that I don't understand something, so if you want - send me a message in Discord and we can have a discussion there 😊